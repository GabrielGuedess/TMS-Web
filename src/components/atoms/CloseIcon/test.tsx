import { render } from '@testing-library/react';

import { CloseIcon } from '.';

describe('<CloseIcon />', () => {
  it('should render the close icon correctly', () => {
    const { container } = render(<CloseIcon />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
