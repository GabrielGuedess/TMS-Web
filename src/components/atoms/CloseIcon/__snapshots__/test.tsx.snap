// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CloseIcon /> should render the close icon correctly 1`] = `
<svg
  fill="none"
  height="24"
  viewBox="0 0 16 16"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M1 1L15.2418 15.0418"
    stroke="currentColor"
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-width="1.2"
  />
  <path
    d="M15.2417 1L0.999904 15.0418"
    stroke="currentColor"
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-width="1.2"
  />
</svg>
`;
